src/main/java/com/rapid7/ias/bamboo/impl/InsightAppSecScanTaskConfigurator.java

```diff
@@ -1,5 +1,6 @@
 package com.rapid7.ias.bamboo.impl;
 
+import aQute.bnd.http.HttpClient;
```

Comment:
Did you mean to import this http client? - Check for unused imports and remove them